types/defer-promise/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Is there a better way to express the following, or an alternative way to disable this linting rule?

I think the exception to no-unnecessary-generics generics is justified. The function `defer()` makes a new `Deferred<T>`, not just any `T`. It's similar to the constructors `new Promise<T>()` or `new Map<T, U>()`.